runhouse/servers/obj_store.py

```diff
@@ -58,6 +84,19 @@ def call_kv_method(store, method, *args, **kwargs):
         else:
             return getattr(store, method)(*args, **kwargs)
 
+    def get_access_level(self, token: str, resource_uri: str):
+        return self.call_auth_cache_method(
+            self._auth_cache, "lookup_access_level", token, resource_uri
+        )
+
+    def add_user(self, token: str):
```

Comment:
I think we want to call add_user from inside the http_server, so we don't need to pass the token through. You can load the AuthCache actor by name and call add_user on it directly with ray.remote and ray.get (you don't need the helper method or anything like that).